android/src/main/java/io/customer/reactnative/sdk/messagingpush/RNCIOPushMessaging.kt

```diff
@@ -136,9 +145,34 @@ class RNCIOPushMessaging(
             }
             true // as this permission listener can be removed now
         }
+
         else -> false // desired permission not yet granted, so we will keep the listener
     }
 
+    override fun onActivityResult(
+        activity: Activity?,
+        requestCode: Int,
+        resultCode: Int,
+        intent: Intent?,
+    ) {
+        // Nothing required here
+    }
+
+    override fun onNewIntent(intent: Intent?) {
```

Comment:
lets maybe add a comment explaing why we have added this

Reply:
[Added](https://github.com/customerio/customerio-reactnative/pull/156/commits/39a5c3e144390f17d9c812e18cad39fff8262f80) comments